gap/Derivations.gd

```diff
@@ -1,63 +1,367 @@
+#############################################################################
+##
+##                                               CAP package
+##
+##  Copyright 2013, Sebastian Gutsche, TU Kaiserslautern
+##                  Sebastian Posur,   RWTH Aachen
+##
+#! @Chapter Managing derived methods
+#!
+##
+#############################################################################
+
+
+####################################
+##
+#! @Section Info class
+##
+####################################
+
+#! @Description
+#!  Info class for derivations.
 DeclareInfoClass( "DerivationInfo" );
+
+#!
 DeclareGlobalFunction( "ActivateDerivationInfo" );
+
+#!
 DeclareGlobalFunction( "DeactivateDerivationInfo" );
 
+
+####################################
+##
+#! @Section Derivation objects
+##
+####################################
+
+#! @Description
+#!  A derivation object describes a derived method.
+#!  It contains information about which operation the derived method
+#!  implements, and which other operations it relies on.
 DeclareCategory( "IsDerivation", IsObject );
```

Comment:
The word "derivation" can mean many things, and I agree that "IsDerivation" is not a good name for this category.  Maybe it could be called "IsDerivedMethod" instead, or something similar to that?

However, I don't understand the sentence "GAP categories are technical types, so they should be given technical names."  Do you mean that the names of GAP categories should in general never contain mathematical terms?

In any case, I don't see how this discussion is relevant for the pull request, since it is about a name that was already there before these commits.